src/components/notifications/NotificationsMenu.js

```diff
@@ -201,6 +214,27 @@ function NotificationsMenu(props) {
                 >
                     {t("notifications")}
                 </Typography>
+
+                {!userProfile?.id && (
+                    <ListItem>
+                        <ErrorTypography
+                            errorMessage={errorMessage}
+                            onDetailsClick={() => setErrorDialogOpen(true)}
+                        />
+                        <DEErrorDialog
+                            open={errorDialogOpen}
+                            id={build(
+                                ids.BASE_DEBUG_ID,
+                                ids.SIGN_IN_ERR_DIALOGUE
+                            )}
+                            errorObject={errorObject}
+                            handleClose={() => {
+                                setErrorDialogOpen(false);
+                            }}
+                        />
```

Comment:
We also have a convenience component that wraps these in `components/utils/error/ErrorTypographyWithDialog`:
https://github.com/cyverse-de/sonora/blob/f028685b75c40ed7f48bb254a7d23eed55a46256/src/components/utils/error/ErrorTypographyWithDialog.js#L16-L35

I would suggest using that component, since it also tracks the fact that this user encountered this sign-in error message (unless we don't want to track that for some reason).